torch/distributed/rpc/api.py

```diff
@@ -350,6 +351,25 @@ def _validate_rpc_args(backend, store, name, rank, world_size, rpc_backend_optio
             )
 
 
+T = TypeVar("T")
+GenericWithOneTypeVar = Generic[T]
+
+
+try:
+    class RRef(PyRRef, GenericWithOneTypeVar):
```

Comment:
Do we need to move all our docs here?

Reply:
Would I be correct if I assume we only need this for typing? Curious, can this be done through pybind? so that we don't need both PyRRef and RRef?

Reply:
> Do we need to move all our docs here?

Do parent class methods got linked to doc?

> Would I be correct if I assume we only need this for typing? Curious, can this be done through pybind? so that we don't need both PyRRef and RRef?

I searched pybind doc a bit. It allows adding metaclass, but I didn't see inheritance. I think the reason is a C++ class (PyRRef) is in a different memory layout.

Reply:
Discussed offline, we will need to add `:inherited-members:` to RRef in `rpc.rst`. 

Then let's verify the docs are still correct by building it locally.